modules/db/src/main/Handlers.scala

```diff
@@ -31,8 +31,8 @@ trait Handlers:
   )(using NotGiven[NoBSONReader[T]]): BSONReader[T] with
     def readTry(bson: BSONValue) = reader.readTry(bson).map(sr.apply)
 
-  given NoBSONReader[Blurs] with {}
-  given NoBSONWriter[Blurs] with {}
+  // given NoBSONReader[Blurs] with {}
```

Comment:
this is a mistake, someone pls uncomment these two lines (I'm sleeping atm) or I'll do it tomorrow. 

without these lines, it compiles but it'll used opaque instances if We don't import blurs.given.

Reply:
But tbh, I'm not sure why there was (or is) a bug in prod. I haven't ran this to make sure it works yet. I'll do it tomorrow if no one beat me to it.

Reply:
answered in: https://github.com/lichess-org/lila/pull/15755#issuecomment-2241396476